README.md

```diff
@@ -1121,6 +1121,16 @@ completed the compilation successfully before loading a view.
 }
 ```
 
+##### webpack or webpack-dev-server not found
+
+* `webpack` or `webpack-dev-server` binstub is not found because webpacker is not installed correctly. This could happen when you skip the `webpacker:install` step. Therefore, please run `bundle exec rails webpacker:install` to finish the installation. 
+
+* If you find such errors occurring on heroku, you might be missing the buildpacks. To set the buildpacks, type these commands in your terminal:
+
+```
+heroku buildpacks:set heroku/ruby
```

Comment:
@ytbryan This is not needed - it's set by default when you deploy a ruby/rails app